frontend/public/components/storage/create-pvc.tsx

```diff
@@ -64,7 +82,14 @@ export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVC
   };
 
   handleStorageClass = storageClass => {
-    this.setState({ storageClass: _.get(storageClass, 'metadata.name'), storageClassAccessMode: _.get(storageClass, ['metadata', 'annotations', 'storage.alpha.openshift.io/access-mode'])}, this.onChange);
+    //if the provisioner is unknown or no storage class selected, user should be able to set any access mode
+    const modes = storageClass && storageClass.provisioner && provisionerAccessModeMapping[storageClass.provisioner] ? provisionerAccessModeMapping[storageClass.provisioner] : ['ReadWriteOnce', 'ReadWriteMany', 'ReadOnlyMany'];
+    //setting message to display for various modes when a storage class of a know provisioner is selected
+    const displayMessage = provisionerAccessModeMapping[storageClass.provisioner] && storageClass ? 'Access mode is set by storage class and cannot be changed.': 'Permissions to the mounted drive.';
```

Comment:
You need to reverse this check or it blows up if `storageClass` is not defined.
```suggestion
    const displayMessage = storageClass && provisionerAccessModeMapping[storageClass.provisioner] ? 'Access mode is set by storage class and cannot be changed.': 'Permissions to the mounted drive.';
```

Reply:
ack